### PR TITLE
Add Bouyant block on home page

### PR DESCRIPTION
**Pull request by dered-dev**
**ADD**: New block for "Bouyant" in home page
**ADD**: Prefix css properties
**Update**: Stats list in home page
**FIX**: Alignment columns in support-training page
**Update**: Dropdown menu "Community"
**Update**: "Buoyant" writing and rename files

### CODE REVIEW

**Review by kflynn — CHANGES_REQUESTED** [review]
There are actually many places beyond William's note where "b**uo**yant" is spelled "b**ou**yant" instead, including filenames and such; we need to get all of those corrected so that we don't all get very confused down the road. 🙂 

Looks good other than that, though! 🙂

File: linkerd.io/layouts/partials/homepage/cta-buoyant.html [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+  <div class="container pt-6 pb-6 cta-bouyant">
+    <div class="container columns">
+      <div class="column">
+        <h2 class="title is-size-1 has-text-color cta-bouyant-title">{{ .title }}</h2>
+        <h5 class="subtitle is-size-4 has-text-color cta-bouyant-subtitle">{{ .subtitle }}</h5>
+      </div>
+    </div>
+    <div class="container">
+      <div class="has-text-centered pt-4 block-created-by">
+        <p class="has-text-weight-medium has-text-color">Created by</p>
```
Comment (wmorgan) [inline linkerd.io/layouts/partials/homepage/cta-buoyant.html:11]:
```suggestion
        <p class="has-text-weight-medium has-text-color">Linkerd was created by</p>
```

File: linkerd.io/layouts/partials/homepage/cta-buoyant.html [hunk 2]
```diff
@@ -0,0 +1,25 @@
+<div class="wrapper has-background-white ">
+  <div class="container pt-6 pb-6 cta-bouyant">
+    <div class="container columns">
+      <div class="column">
+        <h2 class="title is-size-1 has-text-color cta-bouyant-title">{{ .title }}</h2>
+        <h5 class="subtitle is-size-4 has-text-color cta-bouyant-subtitle">{{ .subtitle }}</h5>
+      </div>
+    </div>
```
Comment (wmorgan) [inline linkerd.io/layouts/partials/homepage/cta-buoyant.html:8]:
```suggestion
```

File: linkerd.io/layouts/partials/homepage/stats-list.html [hunk 3]
```diff
@@ -3,15 +3,15 @@
...
+          6+
         </div>
         <div class="is-size-6">
           Years in production
         </div>
       </div>
       <div class="column has-text-centered has-text-white is-half-mobile stats-list-item">
         <div class="is-size-1">
-          7,500+
+          8,000+
```
Comment (wmorgan) [inline linkerd.io/layouts/partials/homepage/stats-list.html:14]:
```suggestion
          8,500+
```

**Review by wmorgan — COMMENTED** [review]
After staring at this for a while, I think we should just remove the CTA text entirely.

**Comment by dered-dev** [comment]
@wmorgan Updated the "Linkerd was created by" block and the stats on home page

**Comment by kflynn** [comment]
Forced DCO to pass since I know it's OK for @dered-dev's commits here; we'll try to sort it out for next time. 🙂

**Review by kflynn — APPROVED** [review]
Looks OK from here, thanks! 🙂

**Comment by wmorgan** [comment]
I'd like to remove the cta-buoyant-title and cta-buoyant-subtitle entirely.

**Review by kflynn — APPROVED** [review]
Looks sane from here.

**Comment by kflynn** [comment]
Forcing DCO and merging.

**Pull request merged**